Userland/Libraries/LibC/CMakeLists.txt

```diff
@@ -234,7 +234,7 @@ endforeach()
 add_library(crt0 STATIC crt0.cpp)
 target_link_libraries(crt0 PRIVATE NoCoverage)
 add_custom_command(
-    TARGET crt0
+    TARGET crt0 POST_BUILD
```

Comment:
Can you say why in the commit description?

As far as I know, cmake implements postbuilds (at least in its ninja generator) but basically tacking them on to the main target's command with a `&& post_build_step`, in the same build rule. So if you happen to cancel the build after the main command but before the build step, things look up to date from an input / output mtime point of view. I think newer versions of ninja do keep enough state in the build log to still realize it needs to rerun the step, but it's imho a bit yucky, and also worse for build parallelism. I try to not use postbuilds if I can help it (by having a real separate build step that touches a stamp file, for example). If we can't help it here, it'd be good to know why :)

Reply:
Without looking into it much, I believe the warning I was getting was saying "hey, this is implicitly a post build, please specify"

Reply:
If we want a distinct separate custom command, then using TARGET is never going to achieve that

Reply:
I added a blurb to the second commit about the CMake policy. The change I made is behavior-preserving.

Reply:
Thanks, great explanation.